core/wiki/macros/tabs.tid

```diff
@@ -4,13 +4,13 @@ tags: $:/tags/Macro
 \define tabs(tabsList,default,state:"$:/state/tab",class,template)
 <div class="tc-tab-set $class$">
 <div class="tc-tab-buttons $class$">
-<$list filter="$tabsList$" variable="currentTab"><$set name="save-currentTiddler" value=<<currentTiddler>>><$tiddler tiddler=<<currentTab>>><$button set=<<qualify "$state$">> setTo=<<currentTab>> default="$default$" selectedClass="tc-tab-selected" tooltip={{!!tooltip}}>
+<$list filter="$tabsList$" variable="currentTab"><$set name="save-currentTiddler" value=<<currentTiddler>>><$set name="tab-class" filter="[<currentTab>get[tab-class]]"><$tiddler tiddler=<<currentTab>>><$button set=<<qualify "$state$">> setTo=<<currentTab>> default="$default$" class=<<tab-class>> selectedClass="tc-tab-selected" tooltip={{!!tooltip}}>
```

Comment:
Imo set isn't needed just use: `class={{!!tab-class}}` in the button. I did test this yesterday and it worked.